### PR TITLE
{math}[GCC/13.2.0,gfbf/2023b] FLINT v3.1.1, NTL v11.5.1

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/n/NTL/NTL-11.5.1-GCC-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+description = """NTL is a high-performance, portable C++ library providing data structures and
+algorithms for manipulating signed, arbitrary length integers, and for vectors,
+matrices, and polynomials over the integers and over finite fields."""
+
+toolchain = {'name': 'GCC', 'version': '13.2.0'}
+toolchainopts = {'pic': True}
+
+github_account = 'libntl'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
```
Comment (Micket) [inline easybuild/easyconfigs/n/NTL/NTL-11.5.1-GCC-13.2.0.eb:21]: FlexiBLAS patch still looks very much necessary.

**Review by Micket — DISMISSED** [review]

File: easybuild/easyconfigs/n/NTL/NTL-11.5.1-GCC-13.2.0.eb (hunk 1 above)
Comment (PetrKralCZ) [reply easybuild/easyconfigs/n/NTL/NTL-11.5.1-GCC-13.2.0.eb:21]: Sorry, I don't understand what you mean. FlexiBLAS patch for `NTL`?

Comment (Micket) [reply easybuild/easyconfigs/n/NTL/NTL-11.5.1-GCC-13.2.0.eb:21]: Yes. The one which is present in the previous easyconfig of this.

Comment (PetrKralCZ) [reply easybuild/easyconfigs/n/NTL/NTL-11.5.1-GCC-13.2.0.eb:21]: I don't see such patch in any of the previous easyconfigs:
- https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/n/NTL/NTL-11.5.1-GCC-12.2.0.eb
- https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/n/NTL/NTL-11.5.1-GCC-11.3.0.eb
- https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/n/NTL/NTL-11.5.1-GCC-11.2.0.eb
- https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/n/NTL/NTL-11.3.4-GCC-8.2.0-2.31.1.eb

Comment (Micket) [reply easybuild/easyconfigs/n/NTL/NTL-11.5.1-GCC-13.2.0.eb:21]: Sorry, i misread eb --review-pr, this was supposed to be for FLINT

Comment (PetrKralCZ) [reply easybuild/easyconfigs/n/NTL/NTL-11.5.1-GCC-13.2.0.eb:21]: Those modifications are now handled by `sed` in `preconfigopts` of the `FLINT` easyconfig.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 0 out of 1 (2 easyconfigs in total)
b-cn1603.hpc2n.umu.se - Linux Ubuntu 22.04, x86_64, AMD EPYC 7313 16-Core Processor, 1 x NVIDIA NVIDIA A100 80GB PCIe, 545.29.06, Python 3.10.12
See https://gist.github.com/akesandgren/0539ab444afd0d2afab4dc0d3aa063ae for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 0 out of 1 (2 easyconfigs in total)
b-cn1603.hpc2n.umu.se - Linux Ubuntu 22.04, x86_64, AMD EPYC 7313 16-Core Processor, 1 x NVIDIA NVIDIA A100 80GB PCIe, 545.29.06, Python 3.10.12
See https://gist.github.com/akesandgren/80fd7144c3d06118cfc2629b339a3f27 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 4 out of 4 (2 easyconfigs in total)
b-cn1603.hpc2n.umu.se - Linux Ubuntu 22.04, x86_64, AMD EPYC 7313 16-Core Processor, 1 x NVIDIA NVIDIA A100 80GB PCIe, 545.29.06, Python 3.10.12
See https://gist.github.com/akesandgren/d569bd10a77b287d20c24fa0bc378bd1 for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**